pumpkin/src/server/mod.rs

```diff
@@ -127,6 +130,35 @@ impl Server {
         }
     }
 
+    /// Sends a message to all players in every world
+    pub fn broadcast_message(&self, content: TextComponent) {
+        for world in &self.worlds {
+            world.broadcast_message(&content);
+        }
+    }
+
+    /// Get all online players
+    pub fn get_online_players(&self) -> impl Iterator<Item = Arc<Player>> + '_ {
+        self.worlds.iter().flat_map(|world| world.get_players())
+    }
+
+    /// Gets the nearest player from the world position
+    pub fn get_nearest_player(&self, target: &Vector3<f64>) -> Option<Arc<Player>> {
+        // TODO respect which world the player is in
```

Comment:
You can easily get the World from a player by calling `player.living_enitity.entity.world`. But i the current plan is to search for Player on all world (e.g. Gamemode, Kick Command...)

Reply:
In minecraft, the nearest player has to be in the same world. If that's the plan though, then I can change it